tests/test_ECCC_forecast.py

```diff
@@ -39,18 +39,9 @@ def test_forecasting_GEPS(self, tmpdir):
         # Extract the final states that will be used as the next initial states
         rvc = model.outputs["solution"]
 
-        # Collect the most recent forecast data for location and climate model.
-        # Limited to GEPS for now
-        with open(get_local_testdata("watershed_vector/LSJ_LL.zip"), "rb") as f:
-            with ZipMemoryFile(f.read()) as z:
-                region_coll = z.open("LSJ_LL.shp")
-                fcst = forecasting.get_recent_ECCC_forecast(
-                    region_coll, climate_model="GEPS"
-                )
-
-        # write the forecast data to file
-        ts = f"{tmpdir}/fcstfile.nc"
-        fcst.to_netcdf(ts)
+        # Collect test forecast data for location and climate model.
+        fcst_fn = get_local_testdata("eccc_forecasts/geps_watershed.nc")
+        fcst = xr.open_dataset(fcst_fn)
```

Comment:
I would suggest to rename `fcst_fn` to `ts20`, to put emphasis on the fact that it's a 20 channel time series (as opposed to Salmon, used above) and I would get rid ot the opened dataset (`fcst`) entirely since it's only used to set the `nc_index` below in the model call. I think 20 should be simply set as a variable (`n_stations = 20` or something) to be used in the asserts.

Reply:
ok